cime/config/e3sm/config_grids.xml

```diff
@@ -1137,6 +1137,46 @@
       <mask>tx0.1v2</mask>
     </model_grid>
 
+    <model_grid alias="ne512np4_360x720cru_ne512np4">
```

Comment:
Do you know what cru is for?

Reply:
Climate Research Unit, which is apparently the source for some of the land forcing data. The 360x720cru grid is described as the "Exact half-degree CRUNCEP datm forcing grid with CRUNCEP land-mask". According to the NCAR Research Data Archive (https://rda.ucar.edu/datasets/ds314.3/#!description), "The CRUNCEP is an atmospheric forcing dataset used forcing the land surface models. Specifically this dataset is designed to drive the Community Land Model over a long period. The CRUNCEP is a combination of two existing datasets; the CRU TS3.2 0.5 X 0.5 monthly data covering the period 1901 to 2002 and the NCEP reanalysis 2.5 X 2.5 degree 6-hourly data covering the period 1948 to 2016."